chart/values.yaml

```diff
@@ -396,7 +399,7 @@ loki-stack:
           - mountPath: /data
             name: storage
     config:
-      # Compactor is a MayastorDB(loki term) Shipper specific service that reduces the index
+      # Compactor is a BoltDB(loki term) Shipper specific service that reduces the index
```

Comment:
Should we refer as Bolt in openebs ?

Reply:
BoltDB is the actual name of the loki service.

Reply:
ok

Reply:
Can we not change to MayastorDB here and wherever applicable?

Reply:
Ok, ignore my above comment. BoltDB seems to be a Grafana term !!!